common/Reducers.jsm

```diff
@@ -634,6 +634,7 @@ function DiscoveryStream(prevState = INITIAL_STATE.DiscoveryStream, action) {
           return Object.assign({}, item, {
             open_url: action.data.open_url,
             pocket_id: action.data.pocket_id,
+            badge: "pocket",
```

Comment:
why badge and not type as used in Sections at.PLACES_SAVED_TO_POCKET handler

Reply:
As discussed offline its because type is reserved in DS to describe the component (i.e "Hero", "CardGrid"). Should we try to do a refactor after rewriting type => component_type and badge => type?

Reply:
We can leave type in reducers.jsm as is, those context menu handlers should be removed with AS experience going away. I propose renaming badge to context_type thats closer to value it stores, wdyt?